src/components/shared/headers/AppHeader.js

```diff
@@ -10,7 +10,7 @@ class AppHeader extends Component {
   headerLinkClasses(route) {
     var classes = ['header-nav-link'];
     
-    if (this.props.location.pathname == route) {
+    if (this.props.location.pathname === route) {
```

Comment:
`===` checks type as well, and since `this.props.location.pathname` and `route` are both guaranteed to be strings, we should only need `==`.